src/api/api.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line
+const BASE_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=d99f9187&t=';
```

Comment:
```suggestion
const BASE_URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=d99f9187&t=';
```
Try finding a movie (using your demo link, not npm start) and open the network tab on devtools. You'll see the problem that I've faced and the solution to this was suggested by Kiril Ivanchenko) so here's a quick fix)